src/framework/components/gsplat/system.js

```diff
@@ -105,6 +126,34 @@ class GSplatComponentSystem extends ComponentSystem {
     onRemove(entity, component) {
         component.onRemove();
     }
+
+    /**
+     * Gets the GSplat material used by unified GSplat rendering for the given camera and layer.
+     *
+     * Returns null if the material hasn't been created yet. In unified mode, materials are created
+     * during the first frame update when the GSplat is rendered. To be notified immediately when
+     * materials are created, listen to the 'material:created' event on GSplatComponentSystem:
+     *
+     * @example
+     * app.systems.gsplat.on('material:created', (material, camera, layer) => {
+     *     // Material is now available
+     *     material.setParameter('myParam', value);
+     * });
```

Comment:
This block needed to go below the param tags.

Reply:
It is. It's more a consistency thing. Every single JSDoc block we have in the engine is in this order:

* Description (no-tag)
* Params (if any)
* Example (if any)